### PR TITLE
Getting started with Grafana and MS SQL

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
1. Added new topic
2. Updated TOC
3. Changes to image alignment in the "Getting Started with Prometheus" topic.

### CODE REVIEW

File: docs/sources/getting-started/getting-started-sql.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+`Database`     | Name of your MS SQL database.
+`User`         | Database user's login/username.
+`Password`     | Database user's password. 
+
+For installations from the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository, `gdev-mssql` data source is available. Once you add this datasource, the `Datasource tests - MSSQL` dashboard with three panels showing metrics generated from test database is available. 
+
+<img src="/img/docs/getting-started/gdev-sql-dashboard.png" class="no-shadow" width="700px">
+
+Use this to quickly spin up the dashboard and edit queries using SQL syntax. Also experiment with the dashboards to customize them to your needs to: 
+- createdifferent panels
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/getting-started/getting-started-sql.md:79]: Capitalize the first letter of each list item, add periods to the end, and follow the list with a blank line.

File: docs/sources/getting-started/getting-started-sql.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+   ```
+    make devenv sources=mssql
+   ```
+This creates an image of the SQL Server database and runs it as a Docker container.
+
+## Step 4: Adding the MS SQL data source
+
+To add MS SQL Server data source:
+
+1. In the Grafana side menu, go to  **Configuration** and click **Data Sources** option.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:61]: Apply repetitive numbering.

Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:61]:
```suggestion
1. In the Grafana side menu, hover your cursor over the **Configuration** (gear) icon and then click **Data Sources**.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana.
+- Log in to Grafana.
+- Create your first dashboard.
+
+## Step 2: Download MS SQL Server
+
+MS SQL Server can be installed on many different operating systems. Refer to the [MS SQL Server downloads page](https://www.microsoft.com/en-us/sql-server/sql-server-downloads), for a complete list of all available options.
+
+Alternately, install MS SQL Server using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository (recommended). Here you will find a collection of supported data sources, including MS SQL Server, along with test data and pre-configured dashboards for use.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:26]:
```suggestion
Alternately, if you are working on Mac or Linux, then you can install MS SQL Server using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository (recommended). Here you will find a collection of supported data sources, including MS SQL Server, along with test data and pre-configured dashboards for use.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+> **Note:** Installing MS SQL Server on Windows from the [grafana/grafana](https://github.com/grafana/grafana/tree/master/devenv) GitHub repository is not supported at this time.
+
+## Step 3: Install MS SQL Server
+
+You can install MS SQL Server on the host running Grafana or on a remote server. To install the software from the [downloads page](https://www.microsoft.com/en-us/sql-server/sql-server-downloads), follow their setup prompts.
+
+Otherwise, follow the instructions below to install and configure MS SQL Server from the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository.
+
+1. Clone the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository to your local system.
+
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:37]: Why are there empty lines between each list step?

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/getting-started/getting-started-sql.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:38]: Removed empty lines.

Comment (oddlittlebird) [reply docs/sources/getting-started/getting-started-sql.md:38]: Weird. It's not showing that on my screen.

File: docs/sources/getting-started/getting-started-sql.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    make devenv sources=mssql
+   ```
+This creates an image of the SQL Server database and runs it as a Docker container.
+
+## Step 4: Adding the MS SQL data source
+
+To add MS SQL Server data source:
+
+1. In the Grafana side menu, hover your cursor over the **Configuration** (gear) icon and then click **Data Sources**.
+2. Filter by `mssql` and select the **Microsoft SQL Server** option.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:63]:
```suggestion
1. Filter by `mssql` and then select the **Microsoft SQL Server** option.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+3. Click **Add data source** button in the top right header to open the configuration page.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:64]:
```suggestion
1. Click **Add data source** in the top right header to open the configuration page.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+4. Enter the information specified in the table below, then click **Save & Test** at the bottom of the page.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:65]:
```suggestion
1. Enter the information specified in the table below, then click **Save & Test**.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+   
+ Name           | Description
+------------   | -------------
+`Name`         | The data source name. This is how you refer to the data source in panels and queries.
+`Host`         | The IP address/hostname and optional port of your MS SQL instance. If port is omitted, the default 1433 will be used.
+`Database`     | Name of your MS SQL database.
+`User`         | Database user's login/username.
+`Password`     | Database user's password. 
+
+For installations from the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository, `gdev-mssql` data source is available. Once you add this datasource, the `Datasource tests - MSSQL` dashboard with three panels showing metrics generated from test database is available.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started-sql.md:75]:
```suggestion
For installations from the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository, the `gdev-mssql` data source is available. Once you add this data source, the `Datasource tests - MSSQL` dashboard with three panels showing metrics generated from test database is available.
```

File: docs/sources/getting-started/getting-started-sql.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+keywords = ["grafana", "intro", "guide", "started"]
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+weight = 400
++++
+
+# Getting started with Grafana and MS SQL Server
+
+Microsoft SQL Server is a popular relational database management system that is widely used in development and production environments. Grafana v5.1+ ships with a built-in Microsoft SQL Server (MSSQL) data source plugin that allows you to query and visualize data from any Microsoft SQL Server 2005 or newer, including Microsoft Azure SQL Database. 
+
+This topic walks you through the steps to create a series of dashboards in Grafana to display metrics from a MS SQL Server database. You can also configure the MS SQL Server data source on a [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) instance without having to host Grafana yourself.
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:13]: Made changes to the introduction as per suggestion.

**Review by oddlittlebird — APPROVED** [review]

File: docs/sources/getting-started/getting-started-sql.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+title = "With Grafana and MS SQL"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+weight = 400
++++
+
+# Getting started with Grafana and MS SQL Server
+
+Microsoft SQL Server is a popular relational database management system that is widely used in development and production environments. Grafana v5.1+ ships with a built-in Microsoft SQL Server (MSSQL) data source plugin that allows you to query and visualize data from any Microsoft SQL Server 2005 or newer, including Microsoft Azure SQL Database.
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:11]: Updated this section.

File: docs/sources/getting-started/getting-started-sql.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md]: Done.

File: docs/sources/getting-started/getting-started-sql.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:63]: Done.

File: docs/sources/getting-started/getting-started-sql.md (hunk 7 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:65]: Done

File: docs/sources/getting-started/getting-started-sql.md (hunk 8 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:75]: Done

File: docs/sources/getting-started/getting-started-sql.md (hunk 6 above)
Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started-sql.md:64]: Done

**Pull request merged**